opsdroid/loader.py

```diff
@@ -179,6 +179,13 @@ def load_config_file(self, config_paths):
                 break
 
         if not config_path:
+            if os.path.isfile(OLD_CONFIG_PATH):
```

Comment:
I would be keen to break this out into a helper function. Just because `loader.py` is already bloated.

Reply:
I created it in help first but then moved back to loader haha I'll change it back to help.
One question about this, should I check if the DEFAULT_CONFIG_PATH exists before copying the files? I'd assume an exception will be thrown if the dir doesn't exists